### PR TITLE
Improve Network Awareness doc

### PR DESCRIPTION
**Pull request by kul3r4**
#### WHAT
Improve Network Awareness doc by adding links and small rephrasing

#### WHY
To make it easier to understand the role of this library

#### HOW
Added links, reworded 

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: docs/network-awareness.md [hunk 1]
```diff
@@ -12,24 +12,24 @@ dependencies {
...
-- Use Paired Bluetooth connection when it is available.
-- Use Wifi when available and Charging.
+- Use paired Bluetooth connection when it is available.
+- Use Wi-Fi when available and watch is charging.
 - Use LTE if no other network is available.
-- Wifi and Cell may be requested specifically by the application.
+- Wi-Fi and LTE can be requested by the application.
 
-This leads to some suboptimal decisions
+Without an intentional network management, following situations may happen:
```
Comment (yschimke) [inline docs/network-awareness.md:27]: "Without intentional network management" or "Without an intentional network management layer"?

"An" feels weird.

**Review by yschimke — APPROVED** [review]
Nice, thanks.

File: docs/network-awareness.md [hunk 2]
```diff
@@ -64,13 +64,13 @@ public interface NetworkingRules {
 }
 ```
 
-It allow allows logging network usage and visibility into network status.
+The library also allows logging network usage and visibility into network status.
```
Comment (yschimke) [inline docs/network-awareness.md:67]: Do we want to tie back this visibility to something user focused?

Like visually demonstrating why results may be stale? Or that only offline music is available?

Comment (kul3r4) [reply docs/network-awareness.md]: Thanks, PTAL

**Review by yschimke — COMMENTED** [review]
LGTM

**Pull request merged**